### ISSUE TITLE
Add section about assests

### ISSUE THREAD

**Issue by freesig** [nannou-org/guide#27]
I think that the gui won't display text without the assets folder being in the users project so we should add this to the guide.
Does nannou-new do this @mitchmindtree Maybe we should make a decision whether or not to encourage it's use. 
There might be a way to just check for system fonts so that this isn't required.

**Comment by tpltnt** [comment]
You are correct @freesig. This is why my PR #15 mentions the assets folder and `DEFAULT_FONT_PATH`. It took a while for me to figure font handling out which is why I wrote the tutorial to help others avoid my mistakes.
